core/src/Model.cpp

```diff
@@ -22,23 +22,31 @@
 
 namespace Nextsim {
 
-const std::string Model::restartOptionName = "model.init_file";
+template <typename Key, typename Value>
```

Comment:
Could some more comments be added to this file, so that it is easier to understand what it is doing?

Reply:
Done